### PR TITLE
Seedlet: Make it possible to hide the author bio

### LINKED ISSUE #2683: Spearhead: Add support for hiding author bio.
See: https://wordpress.com/support/customizer/content-options/#author-bio

We should do this in Seedlet so all child themes get the benefit

### PR DESCRIPTION
**Pull request by scruffian**
This adds a section to the customizer to turn off the Author bio:

Screenshot 2020-10-26 at 13 52 14

I decided not to add the rest of the Jetpack options as it would be more work. We can add them as and when we need to.

Fixes #2683

### CODE REVIEW

**Comment by kjellr** [comment]
Does this need some CSS paired with it or something? I see the option, but unchecking it has no effect: 

Screen Shot 2020-10-26 at 10 07 46 AM

**Comment by scruffian** [comment]
Sorry, you're right there was a bunch of code missing. This should work now...

**Comment by kjellr** [comment]
Thanks — It's working now, but the design has changed. Before: 

Screen Shot 2020-10-26 at 3 35 35 PM

After:

Screen Shot 2020-10-26 at 3 25 54 PM

Can we more closely replicate what we had before? (I'm not sure if having the image there is new in this PR, but that's what I was seeing on my end. 🤔)

**Comment by scruffian** [comment]
I have updated this so that the styles match as closely to the old way of doing it as possible. There are some constraints as we have to use the markup provided by Jetpack, but hopefully this is good enough:
Screenshot 2020-10-27 at 10 14 17


I also adapted Spearhead and did #2648 at the same time:
Screenshot 2020-10-27 at 10 11 18

**Comment by MaggieCabrera** [comment]
When I try this without the jetpack plugin on my local install I see this:

Screenshot 2020-10-27 at 11 51 53

File: spearhead/assets/sass/_author-bio.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.author-link {
+	font-size: var(--global--font-size-xs);
+	position: relative;
+
+	&:after {
```
Comment (MaggieCabrera) [inline spearhead/assets/sass/_author-bio.scss:5]: Maybe we should make this a mixin since we are starting to reuse it?

Comment (scruffian) [reply spearhead/assets/sass/_author-bio.scss:5]: I'm not sure there are quite a few differences. The position is bottom right not top left for example...

**Comment by kjellr** [comment]
This looks great. Any chance we can add a tiny amount of top padding to the `author-name`? Something like 4px or so should be fine. Once that's in place, this should be all set to go. 

Before|After
---|---
Screen Shot 2020-10-27 at 8 46 40 AM|Screen Shot 2020-10-27 at 8 46 30 AM

**Pull request merged**